pkg/controller/certificates/certificate_controller.go

```diff
@@ -109,8 +111,9 @@ func (cc *CertificateController) Run(ctx context.Context, workers int) {
 	defer utilruntime.HandleCrash()
 	defer cc.queue.ShutDown()
 
-	klog.Infof("Starting certificate controller %q", cc.name)
-	defer klog.Infof("Shutting down certificate controller %q", cc.name)
+	logger := klog.FromContext(ctx)
+	logger.Info("Starting certificate controller", "name", cc.name)
+	defer logger.Info("Shutting down certificate controller", "name", cc.name)
```

Comment:
The name should come from the context logger right?

Reply:
Eventually. But right now contextual logging is alpha and off by default. So to avoid regressions, relevant values have to be repeated in the log calls.

Reply:
🤦